### ISSUE TITLE
[FR]: Inject code to process outside of zygote

### ISSUE THREAD

**Issue by RinLin-NYA** [PerformanC/ReZygisk#54]
### Description

Let Zygiskd32 and Zygiskd64 hook processes that not only produced by zygote32 and zygote64, but any other processes, or at least, produced by init process(non-ART process).

### Reason

The **PROPRIETARY** software Zygisk Next announced that they implemented this feature and has been published a module sample at https://github.com/5ec1cff/ZygiskNextModuleSample 
I think this is a better way to hook processes under new Android 15, and as a free and open-source software, I think we should provide a liberal alternative of this function.

### Confirmations

- [X] This feature is not already implemented.
- [X] I have verified that this is not a duplicate feature request.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ThePedroo** [comment]
If I'm able to do, I'm willing to get that feature, however Zygiskd C99 and improvements to hiding are higher priority than this, so it will take some time for us to start working on it

**Comment by ThePedroo** [comment]
Just an update on this: with compatibility changes to add lower Android support, now ptracer is able to inject libzygisk.so even on processes which `libdl.so` isn't loaded by using the linker, so this will open doors to allow an easier injection on init processes.

**Comment by LindaFerum** [comment]
Hi @ThePedroo ! 

Does this allow to bypass grapheneOS's "secure app spawning" Zygisk breakage/limitation https://github.com/PerformanC/ReZygisk/issues/239
  (and thus get Zygisk functionality on rooted graphene with "secure app spawning" on ) ? 

And if not, is there any chance of this new injection capability extended further to allow "proper Zygisking" on Graphene with secure app spawning "on" ?

**Comment by ThePedroo** [comment]
No. "Secure" app spawning makes so that it does not utilize Zygote. No Zygote, hence no Zygisk

**Comment by kokoromagic** [comment]
Sorry we can't understand ZN module since it's closed source
